src/constants.js

```diff
@@ -94,27 +94,21 @@ const SyncEvents = {
 }
 
 // Utils
-const MAIN_PROFILE_ID = 'allChannels'
+export const MAIN_PROFILE_ID = 'allChannels'
 
 // Width threshold in px at which we switch to using a more heavily altered view for mobile users
-const MOBILE_WIDTH_THRESHOLD = 680
+export const MOBILE_WIDTH_THRESHOLD = 680
 
 // Height threshold in px at which we switch to using a more heavily altered playlist view for mobile users
-const PLAYLIST_HEIGHT_FORCE_LIST_THRESHOLD = 500
+export const PLAYLIST_HEIGHT_FORCE_LIST_THRESHOLD = 500
 
 // YouTube search character limit is 100 characters
-const SEARCH_CHAR_LIMIT = 100
+export const SEARCH_CHAR_LIMIT = 100
 
 // Displayed on the about page and used in the main.js file to only allow bitcoin URLs with this wallet address to be opened
-const ABOUT_BITCOIN_ADDRESS = '1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS'
-
-export {
-  IpcChannels,
-  DBActions,
-  SyncEvents,
-  MAIN_PROFILE_ID,
-  MOBILE_WIDTH_THRESHOLD,
-  PLAYLIST_HEIGHT_FORCE_LIST_THRESHOLD,
-  SEARCH_CHAR_LIMIT,
-  ABOUT_BITCOIN_ADDRESS,
+export const ABOUT_BITCOIN_ADDRESS = '1Lih7Ho5gnxb1CwPD4o59ss78pwo2T91eS'
+
+export const API_DATA_SOURCES = {
+  LOCAL_API: 'local',
+  INVIDIOUS_API: 'invidious'
```

Comment:
Since you named it as `API_DATA_SOURCES`, why include `API` in constants names too?

Reply:
This is not addressed yet